README.md

```diff
@@ -47,6 +47,8 @@
     npm install
     npm start
     ```
+    
+    If you face any issues during the installation, it may be because you're using a new Node version, you can fix that by downgrading to the LTS version (Node 16.x)
```

Comment:
Could you add this under the `3. Start Developing`. And we could update the text sth like this: 

Prerequisites:
Before contributing or adding a new feature, please make sure you have already installed the following tools:

[NodeJs](https://nodejs.org/en/download/) (Works with Node LTS version v16.x.x)
Optional [NVM](https://github.com/nvm-sh/nvm): Switch Node version by using nvm use (on Windows, use nvm use v16.x.x). If this is not installed, run nvm install v16.x.x. 

NOTE : instead of saying 16.x.x it would be nice to write exact node version that you use.